crewai_tools/tools/serper_dev_tool/serper_dev_tool.py

```diff
@@ -15,14 +15,14 @@ class SerperDevTool(BaseTool):
 	description: str = "A tool that can be used to semantic search a query from a txt's content."
 	args_schema: Type[BaseModel] = SerperDevToolSchema
 	search_url: str = "https://google.serper.dev/search"
-	n_results: int = None
+	n_results: int = 10
 
 	def _run(
 		self,
 		search_query: str,
 		**kwargs: Any,
 	) -> Any:
-		payload = json.dumps({"q": search_query})
+		payload = json.dumps({"q": search_query, "num": self.n_results})
```

Comment:
@joaomdmoura , this looks good.

I tried breaking it with values like -1 and 101 and Serper didn't throw back any errors.

It is important to note that Serper caps out at 100 results for a single query. I don't know if you'd be interested in warning the users if they set a num > 100.

Reply:
love it